frontend/src/lib/lemon-ui/LemonBadge/LemonBadge.stories.tsx

```diff
@@ -61,14 +61,15 @@ export const Sizes: StoryFn<typeof LemonBadge> = () => {
 }
 
 export const Status: StoryFn<typeof LemonBadge> = () => {
+    const statuses = ['primary', 'success', 'warning', 'danger', 'muted', 'data']
     return (
         <div className="flex space-x-2 items-center">
-            <span>primary:</span>
-            <LemonBadge content={<IconPlusSmall />} status="primary" />
-            <span>danger:</span>
-            <LemonBadge content={<IconPlusSmall />} status="danger" />
-            <span>muted:</span>
-            <LemonBadge content={<IconPlusSmall />} status="muted" />
+            {statuses.map((status) => (
```

Comment:
updated this since they were missing...

i couldn't use this because it swallows tooltips though 🤷